### ISSUE TITLE
How to fix mixed content warnings when moving to production

### ISSUE THREAD

**Issue by rfestag** [owen2345/camaleon-cms#722]
On my development box, I did everything over HTTP. I built the site more or less as I needed, and just copied the database up to my production instance. Unfortunately, all of my links still try to use http instead of https.

How can I change whatever is cached in the database to be http instead of https? Is there a better way to move to production? Honestly, I'd love to see a feature for just exporting configurations (custom fields/content types, changes to Page/Post, etc). That way I could spin up a new site with no content, but that is ready to go.

**Comment by christianmeyer** [comment]
@rfestag, what about the [Camaleon CMS Plugin to Export and Import](https://github.com/owen2345/camaleon_export_import)?

**Comment by rfestag** [comment]
Thanks for the pointer. I had tried that, but didn't have any luck. A few issues:

1) It threw an error on import:
```
15:50:37 web.1       |   SQL (1.2ms)  INSERT INTO "cama_metas" ("key", "value", "objectid", "object_class") VALUES ($1, $2, $3, $4) RETURNING "id"  [["key", "_default"], ["value", "{\"has_category\":false,\"has_tags\":false,\"has_summary\":true,\"has_content\":true,\"has_comments\":false,\"has_picture\":true,\"has_template\":false,\"has_seo\":true,\"not_deleted\":false,\"has_layout\":false,\"default_layout\":\"\",\"icon\":\"\",\"default_template\":\"\",\"has_single_category\":false,\"has_featured\":false,\"has_parent_structure\":false,\"posts_image_dimension\":\"\",\"posts_thumb_versions\":\"\",\"posts_thumb_size\":\"\",\"is_required_picture\":false,\"contents_route_format\":\"post_of_posttype\",\"default_thumb\":\"\"}"], ["objectid", 21], ["object_class", "PostType"]]
15:50:37 web.1       |    (1.2ms)  COMMIT
15:50:37 web.1       | Unpermitted parameters: :id, :objectid, :parent_id, :count, :status, :metas, :fields
15:50:37 web.1       | Completed 500 Internal Server Error in 805ms (ActiveRecord: 226.2ms)
15:50:37 web.1       | 
15:50:37 web.1       | 
15:50:37 web.1       |   
15:50:37 web.1       | NoMethodError (undefined method `with_indifferent_access' for #<ActionController::Parameters:0x007f08027c0428>):
15:50:37 web.1       |   
15:50:37 web.1       | app/apps/plugins/export_content/export_content_helper.rb:373:in `block in save_field_group'
...
```
2) I don't want to export any content, just settings. There is no option just to do that (trying to do that also throws an error)

**Comment by owen2345** [comment]
What about to replace the old url to the new url using the sql file?
I mean:
- export to myfile.sql
- replace old url into new url in myfile.sql
- import myfile_changed.sql into production DB

**Comment by rfestag** [comment]
Sorry for the delayed response. I actually ran into a few issues all at once.
1) I ended up getting around this by following the same basic instructions for changing domains. I do it in two phases: one to replace the hostname, and after that the schema. I do this because I'm not certain whether all references to my hostname are actually URLs. Not sure if that is necessary or not
2) I wanted to migrate to Postgres (I prefer that over MySQL). I couldn't find a good way to do it, so I just bit the bullet and manually reconfigured everything. At this point, yes, I can just export a database and re-import it, following the instructions for changing domains. That just feels very heavy handed.

Don't get me wrong, I'm loving Camaleon (it has made building the website I'm working on so much easier). It just doesn't seem to have a clean way to take a development system and move settings over to production without a lot of planning ahead (yet).
